cmd/image.go

```diff
@@ -84,16 +84,6 @@ func init() {
 	imageCreateCmd.Flags().StringP("description", "d", "", "Description of the image. [required]")
 	imageCreateCmd.Flags().StringSlice("features", []string{}, "features of the image, can be one of machine|firewall")
 
-	// TODO howto cope with these errors ?
-	// err := imageUpdateCmd.MarkFlagRequired("file")
```

Comment:
This is not good, removing the required attribute to a flag, makes tab-completion worse. Why does error handling not work ?

Reply:
It is a global flag, this flag does not belong to the `imageUpdateCmd`. Tests cannot execute anymore because the `init()` fails. I added a new if-condition to the start of ech update function to ensure this flag gets set.

Reply:
This is not the right solution, if file is actually a global flag, but not required in all subcommands, we need to remove the global flag and put it solely to the subcommands it is required and mark it so.